pkgs/development/python-modules/safe-pysha3/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
```

Comment:
```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, pythonOlder
, unittestCheckHook
}:
```

One per line and you're going to need `unittestCheckHoo` to run the tests.

Reply:
The tests are run by `setuptoolsCheckHook` implicitly.

Reply:
Which is deprecated, the log should tell you as much. We only support explicit testing.

Reply:
Added back.